mmv1/third_party/terraform/services/compute/resource_compute_network_test.go.erb

```diff
@@ -106,6 +107,43 @@ func TestAccComputeNetwork_routingModeAndUpdate(t *testing.T) {
 	})
 }
 
+func TestAccComputeNetwork_numericId(t *testing.T) {
+	t.Parallel()
+
+	var network compute.Network
```

Comment:
TODO: Re add `var suffixName = acctest.RandString(t, 10)` here!

Reply:
knew i had added this already, i think you were looking at an old commit. This was added here: https://github.com/GoogleCloudPlatform/magic-modules/pull/9473/commits/5bb5ac8174f54d465ac4ca3f8e8d3c032d81a317